client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -170,8 +170,13 @@ public void channelUnregistered(ChannelHandlerContext ctx) throws Exception {
         }
         channel.set(null);
         flowIdReplyProcessorMap.forEach((flowId, rp) -> {
-            rp.connectionDropped();
-            log.debug("Connection dropped for flow id {}", flowId);
+            try {
+                log.debug("Connection dropped for flow id {}", flowId);
+                rp.connectionDropped();
+            } catch (Exception e) {
+                // Suppressing exception which prevents all ReplyProcessor.connectionDropped from being invoked.
+                log.warn("Encountered exception invoking ReplyProcessor for flow id " + flowId, e);
```

Comment:
Please, use `{}` for logging arguments in log lines (e.g., `flowId`).

Reply:
done.